package.json

```diff
@@ -7,6 +7,7 @@
     "start": "BASE_HREF=/ node server",
     "test": "npm run test-unit && npm run lint && npm run build",
     "test-unit": "BABEL_ENV=test jest --config ./test/jest.config.json",
+    "update-snapshot": "BABEL_ENV=test jest --config ./test/jest.config.json --updateSnapshot",
```

Comment:
To ensure this stays consistent, it's probably worth changing this to `npm run test-unit -- --updateSnapshot`